torch/csrc/jit/codegen/cuda/python_frontend/python_bindings.cpp

```diff
@@ -263,6 +263,24 @@ void initNvFuserPythonBindings(PyObject* module) {
           py::arg("dtype") = Nvf::DataType::Float,
           py::arg("is_cpu") = false,
           py::return_value_policy::reference)
+      .def(
+          "full",
+          [](nvfuser::FusionDefinition& fd,
+             std::vector<int64_t>& size,
+             nvfuser::Scalar arg,
+             Nvf::DataType dtype) -> nvfuser::Tensor {
+            nvfuser::Tensor output = fd.defineTensor();
+            fd.defineRecord(new nvfuser::FullOpRecord(
+                {fd.recordingState(arg())},
+                {fd.recordingState(output())},
+                size,
+                dtype));
+            return output;
+          },
+          py::arg("size"),
+          py::arg("arg"),
+          py::arg("dtype"),
+          py::return_value_policy::reference)
```

Comment:
`full` should be put under the `ops` with other operations like `view` and `squeeze`.